gef-ui/server/routes/supporting-information.js

```diff
@@ -1,27 +1,50 @@
 const express = require('express');
 const multer = require('multer');
-const { multerFilter } = require('../utils/multer-filter.utils');
+const { multerFilter, formatBytes } = require('../utils/multer-filter.utils');
 const {
-  getSupportingDocuments, postSupportingDocuments, uploadSupportingDocument, deleteSupportingDocument,
+  getSupportingDocuments,
+  postSupportingDocuments,
+  uploadSupportingDocument,
+  deleteSupportingDocument,
 } = require('../controllers/supporting-information/supporting-documents');
 const { validateRole, validateToken, validateBank } = require('../middleware');
+const { FILE_UPLOAD } = require('../constants/file-upload');
 
 const router = express.Router();
 
-const upload = multer({ fileFilter: multerFilter }).single('documents');
-router.get('/application-details/:dealId/supporting-information/document/:documentType', [validateToken, validateBank, validateRole({ role: ['maker'] })], getSupportingDocuments);
-router.post('/application-details/:dealId/supporting-information/document/:documentType', [validateToken, validateBank, multer().array('documents', 20)], postSupportingDocuments);
-router.post('/application-details/:dealId/supporting-information/document/:documentType/upload', [validateToken, validateBank, validateRole({ role: ['maker'] })], (req, res, next) => {
-  // eslint-disable-next-line consistent-return
-  upload(req, res, (error) => {
-    if (!error) {
-      next(); // if there are no errors, then continue with the file upload
-    } else {
+const uploadSingle = multer({ limits: { fileSize: FILE_UPLOAD.MAX_FILE_SIZE }, fileFilter: multerFilter }).single('documents');
+const uploadMultiple = multer({ limits: { fileSize: FILE_UPLOAD.MAX_FILE_SIZE } }).array('documents', 20);
+router.get(
+  '/application-details/:dealId/supporting-information/document/:documentType',
+  [validateToken, validateBank, validateRole({ role: ['maker'] })],
+  getSupportingDocuments,
+);
+router.post(
```

Comment:
not a blocker.  Can we add a blank line between the consts and routes etc please